### PR TITLE
Additional setup for sophos utm

### PR DESCRIPTION
**Pull request by zaro0508**
Add additional resources for sophos utm

Pre setup
* create an SSL certificate (requires manual approval)

Sophos resources
* provide HTTPS access on port 443 with vpn.sagebase.org certificate

Post setup
* setup sagebase.org hosted zone
* setup record set for vpn.sagebase.org

### CODE REVIEW

File: cf_templates/sophos-post-setup.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Description: >-
+  Template to complete sophos utm setup
+Resources:
+  HostedZone:
+    Type: "AWS::Route53::HostedZone"
+    Properties:
+      HostedZoneConfig:
+        Comment: "Hosted zone for sagebase.org"
+      Name: "sagebase.org"
+      VPCs:
```
Comment (xschildw) [inline cf_templates/sophos-post-setup.yml]: I don't think this is needed for a visible zone.

File: cf_templates/sophos-post-setup.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >-
+  Template to complete sophos utm setup
+Resources:
+  HostedZone:
+    Type: "AWS::Route53::HostedZone"
+    Properties:
+      HostedZoneConfig:
+        Comment: "Hosted zone for sagebase.org"
+      Name: "sagebase.org"
```
Comment (xschildw) [inline cf_templates/sophos-post-setup.yml]: Is this the domain name? If so it should be 'vpn.sagebase.org' (I will create the NS records to use its name servers in GoDaddy).

File: cf_templates/sophos-post-setup.yml (hunk 1 above)
Comment (zaro0508) [reply cf_templates/sophos-post-setup.yml]: i think your right

File: cf_templates/sophos-post-setup.yml (hunk 2 above)
Comment (zaro0508) [reply cf_templates/sophos-post-setup.yml]: no, this is the hosted zone name.  i think domain name is specified in dns record resource which is set to "vpn.sagebase.org."

**Comment by zaro0508** [comment]
We have decided to setup "vpn.sagebase.org" and forego SSL setup.  SSL would only be for the Sophos admin console, since we plan to keep it only accessible from behind the VPN and only a few people will be accessing it we don't think encryption is required at this time.

**Comment by zaro0508** [comment]
replace this PR with https://github.com/Sage-Bionetworks/admincentral-infra/pull/19 which only sets up "vpn.sagebase.org"

**Comment by zaro0508** [comment]
looks like we will need an SSL cert after all.  I forgot that the user portal is completely exposed.

**Comment by zaro0508** [comment]
This was for simple AD which we will no longer be doing

**Pull request closed without merging**